docs/audit_report/src/conf.py

```diff
@@ -87,3 +88,7 @@
     'maketitle': r'\custommanualmaketitle',
     **auditinfo.unicode_mappings(),
 }
+
+# -- Options for extensions --------------------------------------------------
+
+todo_include_todos = True
```

Comment:
In case `.. todolist::` does not show up if there are no todos left, even with this set to True, maybe add it somewhere in the beginning of the document?

Reply:
The architecture document has it enabled, and there's no noise in the document if there are no todos. I think, its fine to just keep this enabled. But I'll double-check, thanks for the pointer.

Reply:
Oh wait. You mean to insert `.. todolist::` because that summarizes things. Nice!
I tried it: It doesn't render anything if no todos are left. I'll do that in a follow-up PR and also add a CI check that todos must not be present in `release/*` branches.

Reply:
I added the `.. todolist::` macro in all documents and enabled the `todo` extension everywhere: https://github.com/sehlen-bsi/botan-docs/pull/150/commits/ae132841087c9351c3428db701cc7fe88cceaa15

Introducing a CI-check is future-work.